.travis.yml

```diff
@@ -26,17 +26,16 @@ python:
     - 3.7
     - 3.8
 
-matrix:
+jobs:
   include:
+    # Old nosetests - Remove soon
+    - python: 3.7
+      env:
+        - CHECK_TYPE="nosetests"
     # Basic dependencies only
     - python: 3.5
       env:
         - DEPENDS="-r requirements.txt"
-    # Clean install
-    - python: 3.5
-      env:
-        - DEPENDS=""
-        - CHECK_TYPE=skiptests
```

Comment:
I regret this... The `DEPENDS=""` is doing the work here and is still needed to test that `nibabel` will work when no dependencies are pre-installed.